pkg/plugin/twinmaker/handler.go

```diff
@@ -658,3 +658,11 @@ func (s *twinMakerHandler) GetSessionToken(ctx context.Context, duration time.Du
 
 	return info, err
 }
+
+func (s *twinMakerHandler) GetTimeObjectFromStringTime(timeString *string) *time.Time {
```

Comment:
```suggestion
func getTimeObjectFromStringTime(timeString *string) *time.Time {
```

No need for this to be a member variable

Reply:
Incorporated the review comments